### PR TITLE
Re-enable Tests in CI environment

### PR DESCRIPTION
**Pull request by mustaphaturhan**
- Adapt old tests to new environment.
- Remove @next/link's because we aren't using next anymore.

### CODE REVIEW

File: src/components/forms/login-form.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import gql from 'graphql-tag';
-import NextLink from 'next/link';
+// import { Link as RouterLink } from 'react-router-dom';
```
Comment (msusur) [inline src/components/forms/login-form.js]: this should be deleted I suppose?

File: src/components/forms/login-form.js [hunk 2]
```diff
@@ -79,11 +79,11 @@ function LoginForm({ onSubmit, withTitle }) {
             ))}
 
             <Text textAlign="right" mb={2}>
-              <NextLink href="/account/forgot-password">
-                <Link textAlign="right" color="linkBlue">
-                  Forgot password?
-                </Link>
-              </NextLink>
+              {/* <RouterLink href="/account/forgot-password"> */}
```
Comment (msusur) [inline src/components/forms/login-form.js]: why not delete this ?

File: src/components/ui/settings/verification-settings.js [hunk 3]
```diff
@@ -1,7 +1,8 @@
+import React from 'react';
 import PropTypes from 'prop-types';
 import { Heading, Flex, Box, Text, Button } from '@chakra-ui/core';
 import { CheckCircle, ArrowRight } from 'react-feather';
-import Link from 'next/link';
+// import Link from 'next/link';
```
Comment (msusur) [inline src/components/ui/settings/verification-settings.js]: same?

File: src/components/ui/settings/verification-settings.js [hunk 4]
```diff
@@ -24,19 +25,19 @@ function VerificationSettings({ withTitle, isVerified }) {
             : 'You’re not a verified user at the moment. To create a campaign, you should become a verified user.'}
         </Text>
       </Flex>
+      {/* <Link href="/account/my-account/verification"> */}
```
Comment (msusur) [inline src/components/ui/settings/verification-settings.js]: ditto

File: src/components/ui/verification-success.js [hunk 5]
```diff
@@ -21,16 +20,16 @@ function VerificationSuccess() {
...
-        <Link href="/campaigns/start-campaign">
-          <Button variant="outline" color="linkBlue" type="submit">
-            Start a Draft Campaign
-          </Button>
-        </Link>
+        {/* <Link href="/"> */}
+        <Button mr={4} variant="ghost" color="danger" type="submit">
+          Go to Homepage
+        </Button>
+        {/* </Link> */}
```
Comment (msusur) [inline src/components/ui/verification-success.js]: 💃

File: src/components/forms/login-form.js (hunk 2 above)
Comment (mustaphaturhan) [reply src/components/forms/login-form.js]: i will reactive them when we go back to work with login-form and other old pages. they are necessary links but I didn't check them in client-sided create-react-app so I don't know how to implement our software yet.

File: src/components/forms/login-form.js (hunk 1 above)
Comment (mustaphaturhan) [reply src/components/forms/login-form.js]: we may want to remember what we did before when time has been come.

**Pull request merged**